Sources/Collections/CollectionViews/UICollectionView+CollectionSkeleton.swift

```diff
@@ -41,6 +41,14 @@ extension UICollectionView: CollectionSkeleton {
         reloadData()
     }
     
+    func updateDummyDataSource() {
+        if (dataSource as? SkeletonCollectionDataSource) != nil {
```

Comment:
Actually, it could happen that the user calls `updateSkeleton` instead of `showSkeleton` the first time he wants to show the skeleton views. Maybe because he wants to call always `updateSkeleton` so skeleton layers are initialized (on the first call) or either updated (on consecutive calls).

The `updateSkeleton` now allows initializing or updating the skeleton layers, while `showSkeleton` only creates the layers for the first time.

Maybe could be better to use only the new `updateSkeleton` as the `showSkeleton` function so if the user calls `showSkeleton` it updates the skeleton layers instead of doing nothing?

I haven't tested it, but the good point of `updateSkeleton` is that if you want to update the `isSkeletonable` property of any view, it'll create the skeleton layer for that view. In the other hand, I think that if `isSkeletonable` is disabled, I don't think that the skeleton layer will be deleted. This could be addressed.

Anyway, I think that this method should check whether it has dummy data (`showSkeleton` or `updateSkeleton` has been called previously) and in that case only reload data or whether `showSkeleton` has not been called previously and dummy data should be initialized for the first time.